packages/cli/tests/subjects/multiple-prerendering/index.js

```diff
@@ -12,9 +13,10 @@ export default class App extends Component {
 	render(props) {
 		return (
 			<div id="app">
-				<Router url={props.url} onChange={this.handleRoute}>
+				<Router url={props.url} onChange={this.handleRoute} {...props}>
 					<Home path="/" />
 					<Route66 path="/route66" />
+					<Custom path="/custom" {...props} />
```

Comment:
Ohhhhhhhhhhhhh this needs to be documented...
I was figuring out how to do this for days... and then remembered that we already have this